### DISCUSSION TITLE
Website not working

### DISCUSSION THREAD

**Discussion by ghost** [community/community#21990] [Other Feature Feedback, Questions, & Ideas]
<p>Hey everyone. I have been trying to set up a website with GitHub Pages and it did work when I first made it. Now however, the page won’t work and it redirects to my old webhost website. Also if I try adding the /index.html it doesnt work, it just takes me to a github 404.</p>
<p>Website link: <a href="https://alubjosne.github.io/" rel="nofollow noopener">https://alubjosne.github.io/</a></p>

**Comment by ghost** [comment]
<p>I have tried deleting the repository and re-uploading it. I even went as far as to delete my github account.</p>

**Comment by ghost** [comment]
<p>The link doesn’t redirect now but the website still doesn’t exist to github.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hey <a class="mention" href="/alubjamsejosne">@alubjamsejosne</a>,</p>
<p>I’m able to see your page just fine, are you still having issues?</p>

**Comment by AlubJ** [comment, marked as answer]
<p>Hi Andrea, I have fixed the issues regarding the website not working. All I had to do was reset my browser stuff. Also this is my orginal account because of all of these issues. Thanks for your responce - Alub.</p>

**Comment by Nikhil-Chacharkar** [comment]
<p>Hey, can you help me out with the same issue?</p>
<p>I have a repo in the organization with the name as [organisationname].github.io but I am not able to open the site with a domain like <a href="http://orgname.github.io" rel="nofollow noopener">orgname.github.io</a>.</p>
<p>Instead, it opens like <a href="http://orgname.github.io/%5Breponame%5D" class="inline-onebox-loading" rel="nofollow noopener">orgname.github.io/[reponame]</a></p>

**Comment by alanrysis** [comment]
<p>Same issue here with <a href="http://alanrysis.github.io/" rel="nofollow noopener">http://alanrysis.github.io/</a></p>
<p>Can someone help?</p>
<p>Regards</p>

**Comment by shriyakaneriya** [comment]
<p>A very similar issue with me too. My website has been hosted through Github for over 2 years now, but I am suddendly not able to load my site. </p>
<p>Please help,</p>
<p><a href="http://www.shriyakaneriya.com/" rel="nofollow noopener">http://www.shriyakaneriya.com/</a></p>
<p>Thank You</p>

**Reply by saminsemsar** [reply]
I'm having the same issue.

**Reply by guilhermeweb3225** [reply]
estou tendo o mesmo problema tambem

**Comment by muksiddheswar** [comment]
<p>Although it is not mentioned in the <a href="https://pages.github.com/" rel="nofollow noopener">guide</a>, it seems that it is mandatory to select a theme from settings. The page was rendered immidiately after this. Please refer <a href="https://stackoverflow.com/a/48592535/4168707" rel="nofollow noopener">here</a> for detailed answer.</p>

**Comment by R3alpapi** [comment]
<p>I have the same problem and I don’t understand what you mean by resetting your browser stuff</p>

**Comment by R3alpapi** [comment]
<p>I have the same problem and I don’t understand what you mean by “resetting your browser stuff”</p>

**Comment by Pedro5000** [comment]
<p>This was the right answer for me. Thanks!</p>

**Comment by maheshzemse** [comment]
<p>website not publishing on github.in.</p>
<p>please tel me…</p>

**Comment by RuchiraSheyan** [comment]
<p>Worked for me. Thank you.</p>

**Comment by ghost** [comment]
<p>Even I have the same problem please help<br>
<a href="https://kaul-27.github.io/Family/" rel="noopener nofollow ugc">https://kaul-27.github.io/Family/</a></p>

**Comment by ryanrutten** [comment]
<p>Mine is not working to</p>
<p>The link is: <a href="http://roccoryan.github.io" rel="noopener nofollow ugc">roccoryan.github.io</a></p>

**Comment by roik2500** [comment]
<p>can you explain what you mean “resetting your browser stuff”?</p>

**Comment by lausuarez02** [comment]
<p>Hi! I deployed my react website but i can’t see anything but if i go to the google inspector i can see the components and stuff from my website. What am i doing wrong?</p>
<p>Here the link for the website  <a href="https://lausuarez02.github.io/portFolio1.0/" class="inline-onebox" rel="noopener nofollow ugc">React App</a></p>

**Comment by kotharii** [comment]
<p>somebody help me please this is my website it worked till the time I reloaded the page<br>
<a href="https://kotharii.github.io/robofriends/" rel="noopener nofollow ugc">https://kotharii.github.io/robofriends/</a></p>

**Comment by sudhanshuranjanAki** [comment]
<p>Let us solve your problem we help you deploy built and <a href="https://srchout.com/website-development-company" rel="noopener nofollow ugc">website development</a></p>

**Comment by mire777** [comment]
I cant download any public assets. Whats going on here? Is someting broken?

**Comment by Raghavbhalla907** [comment]
I have used the custom domain but my website is having a problem it is saying typo error but the url and the domain are same no typo error

**Comment by AfomiaG** [comment]
I can't see my website After I have all done Please help me https://Clientserve.github.io
